src/test/java/reposense/git/GitLsTreeTest.java

```diff
@@ -13,12 +19,44 @@
 import reposense.util.SystemUtil;
 
 public class GitLsTreeTest extends GitTestTemplate {
+    private static Method isValidWindowsFilenameMethod;
+
+    @BeforeClass
+    public static void beforeSubClass() throws NoSuchMethodException {
```

Comment:
Method name can be just `beforeClass`

Reply:
This is on purpose as the `@BeforeClass` of the parent class and this sub class are meant to be called without overriding it, so that the `@BeforeClass` of both the parent and sub class will be called.

Also if I override it by changing the method name, I'll need to call the `@BeforeClass` of the parent class as well, 


![image](https://user-images.githubusercontent.com/35756209/72346234-7d55c980-3710-11ea-9d66-23653932433e.png)

![image](https://user-images.githubusercontent.com/35756209/72346243-88105e80-3710-11ea-886c-993bb6ef6275.png)

I'll commit these changes first, then see if this new change is fine